website/content/en/status/report-2023-04-2023-06/opensearch.adoc

```diff
@@ -0,0 +1,14 @@
+=== OpenSearch
+
+Links: +
+link:https://opensearch.org/[OpenSearch] URL: link:https://opensearch.org/[] +
+
+Contact: OpenSearch Team <opensearch@FreeBSD.org>
+
+OpenSearch is a community-driven, open-source search and analytics suite derived from Apache 2.0 licensed ElasticSearch 7.10.2 & Kibana 7.10.2. It consists of a search engine daemon, OpenSearch, and a visualization and user interface, OpenSearch Dashboards. OpenSearch enables people to easily ingest, secure, search, aggregate, view, and analyze data. These capabilities are popular for use cases such as application search, log analytics, and more. With OpenSearch people benefit from having an open-source product they can use, modify, extend, monetize, and resell how they want.
+
+Since our last status report, the OpenSearch Team has regularly updated the OpenSearch and OpenSearch-Dashboards ports to follow upstream releases.
+
+While minor version updates are generally straightforward, some use case make this slightly harder and require more work. For this reason, the OpenSearch Team has recently decided to change the way we package OpenSearch.  Instead of our current scheme where we provide 2 ports, one for the latest major version (`textproc/openseach`, currently at version 2.7.0) and one for the legacy major version in maintenance mode(`textproc/openseach13`, currently at version 1.3.10), we will move to distribute each minor version as a new port (i.e. `textproc/openseach26`, `textproc/openseach27`, `textproc/openseach28` and so on).
+
+Because upstream only provide updates for the latest major version, when a new minor version is made available and a new port is added, the previous minor version will be marked as deprecated, and the minor version before that will have an expiration date set, and will be removed at least three months later.  Users are advised to use the latest minor version, but this will not happen automatically and provide a grace period for users running software that needs to be updated accordingly.
```

Comment:
s/upstream only provide/upstream only provides/

s/this will not happen automatically and provide/this will not happen automatically and provides/